From 1fc37227522ccd314f82ec893ed15c6129296604 Mon Sep 17 00:00:00 2001
From: Paul Jensen <pauljensen@chromium.org>
Date: Wed, 11 Oct 2017 08:37:34 -0400
Subject: [PATCH] Move exclude_unwind_tables back into declare_args

There is desire to adjust this flag manually.

BUG=762629
R=thakis@chromium.org

Change-Id: I3bd134c19270cd1f729b3ea078674e734493d4ab
---

diff --git a/build/config/compiler/compiler.gni b/build/config/compiler/compiler.gni
index ad40fd9..50e19a4 100644
--- a/build/config/compiler/compiler.gni
+++ b/build/config/compiler/compiler.gni
@@ -68,18 +68,18 @@
 
   # Whether or not we should use position independent code.
   use_pic = true
+
+  # Exclude unwind tables for official builds as unwinding can be done from
+  # stack dumps produced by Crashpad at a later time "offline" in the crash
+  # server. For unofficial (e.g. development) builds and non-Chrome branded
+  # (e.g. Cronet which doesn't use Crashpad, crbug.com/479283) builds it's
+  # useful to be able to unwind at runtime.
+  exclude_unwind_tables = (is_chrome_branded && is_official_build) ||
+                          (is_chromecast && !is_cast_desktop_build && !is_debug)
 }
 
 assert(!is_cfi || use_thin_lto, "CFI requires ThinLTO")
 
-# Exclude unwind tables for official builds as unwinding can be done from stack
-# dumps produced by Crashpad at a later time "offline" in the crash server.
-# For unofficial (e.g. development) builds and non-Chrome branded (e.g. Cronet
-# which doesn't use Crashpad, crbug.com/479283) builds it's useful to be able
-# to unwind at runtime.
-exclude_unwind_tables = (is_chrome_branded && is_official_build) ||
-                        (is_chromecast && !is_cast_desktop_build && !is_debug)
-
 # If true, optimize for size. Does not affect windows builds.
 # Linux & Mac favor speed over size.
 # TODO(brettw) it's weird that Mac and desktop Linux are different. We should
